stdlib/internal/debug/debug_test.flux

```diff
@@ -44,3 +44,32 @@ testcase sink {
 
     testing.diff(got, want) |> yield()
 }
+
+testcase get_option {
+    got = debug.getOption(pkg: "internal/debug", name: "vectorize")
+    want = false
+
+    testing.diff(got: array.from(rows: [{v: got}]), want: array.from(rows: [{v: want}]))
+        |> yield()
+}
+
+testcase get_option2 {
+    option debug.vectorize = true
+
+    got = debug.getOption(pkg: "internal/debug", name: "vectorize")
+    want = true
+
+    testing.diff(got: array.from(rows: [{v: got}]), want: array.from(rows: [{v: want}]))
+        |> yield()
+}
+
+testcase get_option_in_map {
+    option debug.vectorize = true
```

Comment:
I did not know that you could use an option statement like this. Normally it has to be at the top of the file. I guess the AST-manipulation for running test does this. This is a nice surprise.